CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+##2016-10-03 - Supported Release 2.0.3
+###Summary
+
+Small release with bugs fixes and another speed improvement.
+
+###Bug Fixes
+- Miscellanous fixes which improve reliability
+- Address json pure version problems ([MODULES-3640](https://tickets.puppetlabs.com/browse/MODULES-3640))
```

Comment:
Remove this one as well. Not something folks using the module tend to care about.